core/core.pro

```diff
@@ -15,7 +15,7 @@ SENSORFW_INCLUDEPATHS = .. \
 DEPENDPATH += $$SENSORFW_INCLUDEPATHS
 INCLUDEPATH += $$SENSORFW_INCLUDEPATHS
 
-DEFINES += PLUGIN_DIRECTORY=\"\\\"$$[QT_INSTALL_LIBS]/sensord-qt5\\\"\"
+DEFINES += PLUGIN_DIRECTORY=\"\\\"/usr/lib/sensord-qt5\\\"\"
```

Comment:
I wonder if we cant somehow just define this QT_INSTALL_LIBS variable, maybe with an envvar or a qmake flag (same comment for recipes-nemomobile/buteo/buteo-mtp_git.bb in https://github.com/AsteroidOS/meta-asteroid/pull/61/